test/cli/test/simple/variable_substitution_globally_without_variable_file/resource.yaml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Why were these original test cases renamed and updated for this PR? Should we instead add new ones?